applications/FluidDynamicsApplication/custom_elements/fluid_element.cpp

```diff
@@ -743,6 +815,8 @@ const ConstitutiveLaw::Pointer FluidElement<TElementData>::GetConstitutiveLaw()
 }
 
 
+
+
```

Comment:
Please avoid adding unnecessary lines to the committed changes.

Reply:
reverted